common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -606,6 +606,12 @@
     ],
     "sqlState" : "42711"
   },
+  "COLUMN_ARRAY_ELEMENT_TYPE_MISMATCH" : {
```

Comment:
The error is pretty specific to JDBC. Should we create sub-class of `FAILED_JDBC`?

Reply:
The scope of the condition is much smaller, and belongs to JDBC only. Let's reflect this fact, and add JDBC to the name.